### ISSUE TITLE
When loading Z-Move info in battle, its PP color is kept from the original move

### ISSUE THREAD

**Issue by AsparagusEduardo** [rh-hideout/pokeemerald-expansion#2410]
![zmovepp](https://user-images.githubusercontent.com/2904965/197348061-596441c5-d019-4f96-81e9-5466f21320db.gif)

**Comment by AlexOn1ine** [comment]
Would this be a decent solution? 
```
    if (!gBattleStruct->zmove.viewing)
        var = GetCurrentPpToMaxPpState(chooseMoveStruct->currentPp[gMoveSelectionCursor[gActiveBattler]],
                         chooseMoveStruct->maxPp[gMoveSelectionCursor[gActiveBattler]]);
    else
        var = GetCurrentPpToMaxPpState(chooseMoveStruct->currentPp[gMoveSelectionCursor[gActiveBattler]], 1);
```
![z_move_pp](https://user-images.githubusercontent.com/93446519/218325428-78ba4a23-ed6d-44ab-ae2b-6fc5cfb94be2.gif)

**Comment by DizzyEggg** [comment]
@AlexOn1ine Yeah, this is a good solution, feel free to submit a PR with your fix.
Or potentially, you could read max PP from the gBattleMoves to support Z-moves which have more than 1 PP(dunno if that's ever the case)

**Comment by DizzyEggg** [comment]
Fixed by #2690
